app/controllers/members_controller.rb

```diff
@@ -18,8 +18,19 @@ def show
     @flickr_auth   = @member.auth('flickr')
     @facebook_auth = @member.auth('facebook')
     @posts         = @member.posts
-    @gardens       = @member.gardens.active.order(:name)
-    @harvests      = @member.harvests
+    plantings = Planting.select(:id, "'planting' as event_type", 'planted_at as event_at', :owner_id, :crop_id, :slug)
```

Comment:
Useful as a standalone performance enhancement separate to this PR?

Reply:
It's actually different data.
I changed the members#show to be a list of activity because getting the existing tabulated display of gardens working again in bootstrap 4 was a big job.